### PR TITLE
Update CONTRIBUTING.md

### LINKED ISSUE #120: Docs: Update `contributing.md` to add a section on the process to follow when you want to open PRs/Issues
Currently the file only contains info on how to run the project locally. Some ideas on sections to include and what info I think should be mentioned:
- **Running the project locally**
  - Cloning the repo
  - Running the backend
  - Running the frontend
- **Opening an issue**
  - Search through the open issues and add one if you don't find it anything related.
  - Search through the closed/merged issues and if an issue you want to open has a merged PR, then reopen that issue with details on why you think the issue was not addressed.
  - Add labels related to your issue
- **Opening a PR**
  - Ask to work on an issue if you want to open a PR related to an issue, that way you can be assigned and others know you're working on it
  - Avoid opening a PR without an accompanying issue, so maybe open an issue to discuss if what you want to add to the project will be welcomed before you write complex features that may or may not be merged
  - Link issues related to your PR via the PR template or the development section on the right sidebar

### PR DESCRIPTION
**Pull request by satyamgupta1495**
## 🛠️ Fixes issue(s)

 Closes #120 

## ✔️ Checklist (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -42,3 +42,22 @@ git checkout -b name-of-my-branch
 git push -u origin name-of-my-branch
 ```
 10. Open a pull request
+
+
+# Running the project locally
```
Comment (Ngoakor12) [inline CONTRIBUTING.md]: this should be a level 2 heading

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -42,3 +42,22 @@ git checkout -b name-of-my-branch
...
+
+1. Cloning the repo
+2. Running the backend
+3. Running the frontend
+
+# Opening an issue
```
Comment (Ngoakor12) [inline CONTRIBUTING.md]: this should be a level 2 heading

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -42,3 +42,22 @@ git checkout -b name-of-my-branch
...
+
+1. Search through the open issues and add one if you don't find it anything related.
+2. Search through the closed/merged issues and if an issue you want to open has a merged PR, then reopen that issue with details on why you think the issue was not addressed.
+3. Add labels related to your issue
+
+# Opening a PR
```
Comment (Ngoakor12) [inline CONTRIBUTING.md]: this should be a level 2 heading

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
You just copy and pasted my whole draft, I though you would add a few points or clean up some of the sentences but it's fine...

Some things to address:
- Looking at the file as a whole, the local setup instructions are duplicated, please have one local setup section
- The subheading should be level 2 headings since contributing on line 1 is already an h1

File: CONTRIBUTING.md (hunk 3 above)
Comment (satyamgupta1495) [reply CONTRIBUTING.md]: I've made the necessary changes plz take a look at it.

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
This still needs some work, information-organisation-wise. I'll get more specific:

The general structure should look like this:
```
# Contributing(h1)
some text(Everyone is welcome to raise is.....)

## Getting setup to contribute(h2)
the info from "1. clone the repo" to "10. open a pull request"

## Opening an issue(h2)
some info about opening an issue

## Opening a PR(h2)
some info about opening a PR
```

**Comment by Ngoakor12** [comment]
@satyamgupta1495  I haven't had a response on this in 4 days. I'm waiting 24 hours to hear back from you, if nothing by then I'm closing the PR  and reassigning the issue

**Comment by Ngoakor12** [comment]
@satyamgupta1495 I'm closing this PR since I'm not getting any response.

**Pull request closed without merging**